### DISCUSSION TITLE
I mapped a million of them

### DISCUSSION THREAD

**Discussion by clawsoon** [tskit-dev/tskit#1747] [Show and tell]
EDIT: Corrected version here:

https://www.youtube.com/watch?v=YbYzvS45vYI&list=PL0elTBjXjW_7oDLwWQCaDxKPSzCus5fAB

I made a 5 part, 42 hour video series out of the non-singleton variants on chromosome 22 out of your data.  If you've got nothing else to do with your life (or you just want to listen to a MIDI harp playing random genetic variants), you can find it here:

https://www.youtube.com/watch?v=6Ev2WgLk9Oc&list=PL0elTBjXjW_52wz-JBuRfFQIgrRUfONWR

Code - not necessarily fit for consumption - is here:

https://github.com/clawsoon/chromosome_movie

**Comment by benjeffery** [comment]
Wow! Cool to see the data like this, thanks for sharing.

**Reply by clawsoon** [reply]
Thanks!

**Reply by clawsoon** [reply]
Corrected version here, with double-counting hopefully eliminated:

https://www.youtube.com/watch?v=YbYzvS45vYI&list=PL0elTBjXjW_7oDLwWQCaDxKPSzCus5fAB

I've switched from mapping mutation events to mapping alleles, with the date given being that of the earliest estimated mutation event for that allele.

One interesting consequence of switching from "only include a _mutation_ if it's found in more than one person" to "only include an _allele_ if it's found in more than one person" is that I'm now showing a whole bunch of time=0 mutations.  More than 10% of my total are alleles found in more than one person which were inferred to be the result of independent mutations for each person.

You can see them in the last 3 hours of the 5th video, labelled "Less than 25 years".  Many of them are geographically close (multiple people in Japan, say, or Gambian Mandinka and Yoruba, or San and Ju hoan) in a way which suggests that some (many?) inferred separate mutation events may have been the same mutation events.

**Comment by clawsoon** [comment]
I think I may have screwed up by not realizing that tree.leaves() gives me all leaves under a node, including leaves under branches with subsequent mutations at the same site.  I may have to actually read the documentation and then remake it all, lol...
